### DISCUSSION TITLE
About ATK-M750C

### DISCUSSION THREAD

**Discussion by 785200** [community/community#119898] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why is the 4G initialization successful but the HUAWEI CLOUD device is not online?

**Comment by 785200** [comment]
Originally, it was fine in the morning, but it was not possible in the afternoon, and it can still be initialized, but the device on HUAWEI CLOUD does not show online, it is still offline, and it cannot accept the uploaded data.

**Comment by Akash1134** [comment]
Hi @785200 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
